Polishchuk_Dariia_HW2(1 sum and avarege ).cpp

```diff
@@ -0,0 +1,23 @@
+#include <iostream>
+
+int main() {
+  size_t a_size;
```

Comment:
```suggestion
  size_t a_size{};
```
используйте инициализацию по дефолту, для того чтобы всегда гарантировать что ваша переменная инициализированная